Formula/cherokee.rb

```diff
@@ -47,6 +47,10 @@ def install
     (pkgshare/"admin/server.py").chmod 0755
   end
 
+  test do
+    assert_match "Version: #{version}", shell_output("script -q /dev/null #{sbin}/cherokee -i | cat")
```

Comment:
Why not `shell_output("#{sbin}/cherokee -i")` or `shell_output("#{sbin}/cherokee -i </dev/null")`?

Reply:
I would like to test the version number instead of only startup.